components/mpas-ocean/bld/namelist_files/namelist_defaults_mpaso.xml

```diff
@@ -152,6 +152,13 @@
 <config_Redi_horizontal_ramp_max>30e3</config_Redi_horizontal_ramp_max>
 <config_Redi_horizontal_ramp_max ocn_grid="WCAtl12to45E2r4">40e3</config_Redi_horizontal_ramp_max>
 
+<!-- submesoscale_eddy_parameterization -->
+<config_submesoscale_enable>.true.</config_submesoscale_enable>
```

Comment:
One more comment.  The tags claim that this is a stealth feature but it looks here like it's on by default.  Am I missing something?